### PR TITLE
Fix: add prettier as dev-dependancies

### PR DESCRIPTION
**Pull request by ckrook**
## **Describe pull-request**  
Updating the package.json file so that we store prettier in dev-dependancies instead of dependancies.

## **Issue Linking:**  
[CDEP-3453](https://tegel.atlassian.net/browse/CDEP-3453?atlOrigin=eyJpIjoiNTY4MDY2YzMxZTZlNDgyN2JjYTA3OTg0M2JhOTk5NDMiLCJwIjoiaiJ9)

## **How to test**  
Checkout to this branch and try to spin up the project.

## **Checklist before submission**
- [x] All existing tests pass
- [x] Not breaking production behavior
- [x] `npm run build-all` without errors

## **Additional context**  
The test for table pagination failed due to missmatch in snapshot. Therefore I had to generate new screenshots for that scenario. The component in Storybook looked ok.

[CDEP-3453]: https://tegel.atlassian.net/browse/CDEP-3453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Lunkan89 — COMMENTED** [review]
The changed screenshot has nothing to do with moving prettier?

File: packages/core/package.json [hunk 1]
```diff
@@ -387,10 +387,12 @@
...
     "@popperjs/core": "2.11.8",
     "@stencil/core": "4.19.2",
-    "date-fns": "2.30.0",
-    "prettier": "2.7.1"
+    "date-fns": "2.30.0"
   },
   "devDependencies": {
+    "prettier": "2.7.1",
+    "commitlint": "^15.0.0",
+    "@commitlint/config-conventional": "^16.2.1",
```
Comment (theJohnnyMe) [inline packages/core/package.json:395]: Do we need this here? Can it stay just on the root level of mono repo?

**Comment by theJohnnyMe** [comment]
> The changed screenshot has nothing to do with moving prettier?

It blocks pushing if the test fails, therefore we needed your PR in - I think that was the reason.

File: packages/core/package.json (hunk 1 above)
Comment (theJohnnyMe) [reply packages/core/package.json:395]: Maybe even prettier could be moved to root level too?

**Pull request closed without merging**